erlang.hcl

```diff
@@ -0,0 +1,39 @@
+description = "Erlang/OTP is a concurrent, distributed runtime for building soft real‑time systems."
+homepage = "https://www.erlang.org/"
+provides = ["erlang"]
+test = "erl -version"
+
+platform "darwin" "amd64" {
+  binaries = ["erts-15.2.2/bin/*"]
+  source = "https://github.com/erlef/otp_builds/releases/download/OTP-${version}/otp-x86_64-apple-darwin.tar.gz"
+}
+
+platform "darwin" "arm64" {
+  binaries = ["erts-15.2.2/bin/*"]
+  source = "https://github.com/erlef/otp_builds/releases/download/OTP-${version}/otp-aarch64-apple-darwin.tar.gz"
+}
+
+platform "linux" {
+  strip = 1
+  binaries = ["bin/*"]
+  source = "https://builds.hex.pm/builds/otp/${arch}/ubuntu-22.04/OTP-${version}.tar.gz"
```

Comment:
I don't feel great about pinning this to Ubuntu 22.04, but it's the latest LTS.

Erlang does have some other options for Debian and Rockylinux: https://www.erlang-solutions.com/downloads/

Reply:
Are there arm64 packages? If so can you add those too please? We're going to be enforcing linux/arm64 soon.

Reply:
Oh they are being downloaded! Nice!